proto/observer/tx.proto

```diff
@@ -27,8 +27,8 @@ message MsgUpdateCoreParamsResponse {}
 
 message MsgAddObserver {
   string creator = 1;
-  int64 chain_id = 2;
-  ObservationType observationType = 3;
+  string observer_address = 2;
+  string zetaclient_grantee_pubkey = 3;
```

Comment:
is observer_address the val operator address? 
is the grantee_pubkey the hotkey pubkey?

Reply:
yes